### PR TITLE
Add Image::surface property

### PR DESCRIPTION
**Pull request by nt8r**
This is the only way to set HIDPI-aware image content.

### CODE REVIEW

**Comment by nt8r** [comment]
See [gtk#613](https://gitlab.gnome.org/GNOME/gtk/-/issues/613) for more background.

I'm not certain I've implemented the `PropertyValueCompare`/`PropertyValueCoerce` traits in an idiomatic way, so review would be appreciated.

File: vgtk/src/lib.rs [hunk 1]
```diff
@@ -633,10 +633,15 @@ pub fn start<C: 'static + Component>() -> (Application, Scope<C>) {
...
+    let id_cell = std::rc::Rc::new(std::cell::RefCell::new(None));
+    let id_cell_clone = id_cell.clone();
+    id_cell.replace(Some(app.connect_activate(move |app| {
         debug!("{}", "Application has activated.".bright_blue());
         constructor(());
-    });
+        if let Some(id) = id_cell_clone.replace(None) {
+            glib::signal::signal_handler_disconnect(app, id);
+        }
+    })));
```
Comment (pepijndevos) [inline vgtk/src/lib.rs:645]: What is the purpose of this code?

Comment (nt8r) [reply vgtk/src/lib.rs:645]: This was my fix for the same issue as https://github.com/bodil/vgtk/commit/c6325f8bc43eafd98a32705204e15053a62eba60 addressed, but is (presumably) no longer necessary. I'm updating my fork now.

**Comment by pepijndevos** [comment]
I think the clippy error has nothing to do with your change?

**Pull request merged**